### ISSUE TITLE
Harvest Moon: A Wonderful Life Expansion

### ISSUE THREAD

**Issue by ginnyclaire** [StardewModders/mod-ideas#229]
Hello, hello!

So my nostalgic self is currently working on an expansion mod to make Forget-Me-Not-Valley (from Harvest Moon: a Wonderful Life) a neighboring town to Pelican Town and accessible by bus! I'm aiming to make it as close to the source material as possible with all the same characters (marriage-able and un-marriage-able), schedules, likes/dislikes, and dialogue as the original game. It's a hefty project, I know.

While I've already got a partner working on the outdoor map, I'd love some more creative geniuses to help me with indoor maps and character portraits! My forté is definitely NPCs, writing events/dialogue/schedules and I've been trying my hand at sprites, but I would LOVE for this mod to really be something great!

Anyone willing to help out?

![image](https://user-images.githubusercontent.com/47725357/52920272-5b6d8000-32d0-11e9-9fab-f6bea5cd64b6.png)

**Comment by Opalie** [comment]
Joining officially to make all things related to maps. (interiors and town, etc)

**Comment by Froststar11** [comment]
Aww Opalie beat me to it. But I'd love to collab on maps, especially interiors! This sounds so fun!

**Comment by ginnyclaire** [comment]
Absolutely!! I’d love for you to help!! Y’all just coordinate on who wants to do what! I’ll post a link to the play through I’ve been watching for reference! I think the game actually translates really well to stardew’s style so I’m excited!

**Comment by ginnyclaire** [comment]
https://youtu.be/PBdPN-9ZQIc Okay here it is! I’ll make a group chat in discord too and add whoever else joins!

**Comment by tiakall** [comment]
This is recruiting for an existing (or formerly existing) idea, not an idea up for others to grab, and is outside the scope of the mod ideas github.

**Comment by ribbon-kitten** [comment]
Is this going anywhere? WOULD LOVE to see this, maybe even help to some extent.

**Comment by Dailycomb** [comment]
> Is this going anywhere? WOULD LOVE to see this, maybe even help to some extent.

As it has been years since this was posted, I would like to take this back up. I can code and create art, but would love to split the work up. Are you interested in helping? If anyone else is interested, let me know and we can create a discord group.
